reporting/api/src/main/java/org/jboss/windup/reporting/config/Hint.java

```diff
@@ -178,6 +190,11 @@ public int getEffort()
         return links;
     }
 
+    public Set<String> getTags()
+    {
+        return UnmodifiableSet.decorate(tags);
```

Comment:
Why use apache-commons, when you have

    java.util.Collections.unmodifiableSet(Set<? extends T>)